dashboard/app/models/regional_partner.rb

```diff
@@ -45,6 +45,13 @@ class RegionalPartner < ActiveRecord::Base
   validates :zip_code, us_zip_code: true, if: -> {zip_code.present?}
   validates_inclusion_of :state, in: STATE_ABBR_WITH_DC_HASH.keys.map(&:to_s), if: -> {state.present?}
 
+  # assign a program manager to a regional partner
+  def program_manager=(program_manager_id)
+    # TODO: switch this to use find_create_by, which is tricky with a has_many through association
```

Comment:
If you're going to own this, please add your name in parentheses. Otherwise, is this really something we're going to do?

Alternatively, you could have this take a `User` instead of id, and leave the find vs. create to the caller.